web/viewer.css

```diff
@@ -16,6 +16,8 @@
 @import url(pdf_viewer.css);
 
 :root {
+  --zoom-factor: 1;
```

Comment:
This needs to be placed in `pdf_viewer.css` instead, since otherwise you're breaking the standalone viewer components.